src/routes/list/members/DataTable.js

```diff
@@ -78,37 +134,37 @@ class DataList extends React.Component<Props, State> {
           >
             <Column width={70} align="center" fixed>
               <HeaderCell>Id</HeaderCell>
-              <Cell dataKey="id" />
+              <EditCell onChange={this.handleChange} dataKey="id" />
```

Comment:
should the index(serial number) be editable?

Reply:
@adibas03 good catch. fixed

Reply:
@adibas03 are we good to go with this?